app/views/forms/make_live/confirmation.html.erb

```diff
@@ -0,0 +1,22 @@
+<% set_page_title(t('page_titles.your_form_is_live')) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @form.name %></span>
+    <h1 class="govuk-heading-l"><%= t('page_titles.your_form_is_live') %></h1>
+
+    <p class="govuk-body">
+      The URL for your form is:
+
+      <span class="govuk-!-display-block govuk-!-margin-top-6 govuk-!-padding-bottom-6">
+        <%= govuk_link_to link_to_runner(ENV["RUNNER_BASE"], @form.id ), link_to_runner(ENV["RUNNER_BASE"], @form.id ), { target:"_blank", rel:"noopener noreferrer"} %>
+      </span>
+    </p>
```

Comment:
I'm planning on stripping this out into a proper view component in another piece of work where we will be playing back this content on other pages.

Reply:
Good idea!